Admin's Manual.md

```diff
@@ -121,9 +121,14 @@ discord:
     compact_mode: true # optional: disables avatars so messages take up less vertical space
 ```
 
-#### Using the Twitter integration
+#### Using the X integration
+Devzat supports sending updates about who is online on the X social media platform. However, because of the switch from free to paid API keys, This feature no longer is no longer updated.
```

Comment:
Why do you think it won't be updated? It might be.

Reply:
changed that